packages/api-docs-builder/utils/parseSlotsAndClasses.ts

```diff
@@ -27,99 +30,168 @@ export interface Slot {
   default?: string;
 }
 
-function extractClasses({
+export default function parseSlotsAndClasses({
   project,
   componentName,
+  muiName,
 }: {
   project: TypeScriptProject;
   componentName: string;
-}): { classNames: string[]; descriptions: Record<string, string> } {
-  const result: { classNames: string[]; descriptions: Record<string, string> } = {
-    classNames: [],
-    descriptions: {},
+  muiName: string;
```

Comment:
Would it be possible to use an enum here? Otherwise it's a pain when we navigate this kind of code and don't know exactly which convention is used

Reply:
Could you give an example? Where exactly would you see an enum?

Reply:
I misunderstood what this property is about. I thought it was storing the mui project (core, base, joy, ...) but it's the component name

Reply:
No, it's the name of the component used in style overrides object (such as `MuiButton`).